pyproject.toml

```diff
@@ -31,13 +31,13 @@ classifiers = [
 python = "^2.7 || ^3"
 
 [tool.poetry.dev-dependencies]
-wsgidav = "^3.0.2"
-pytest = "^5.3.2"
-pytest-cov = "^2.8.1"
-pytest-isort = "^0.3.1"
-pytest-flake8 = "^1.0.4"
-twine = "^3.1.1"
-bump2version = "^0.5.11"
+wsgidav = "*"
+pytest = "*"
+pytest-cov = "*"
+pytest-isort = "*"
+pytest-flake8 = "*"
+twine = "*"
```

Comment:
Is there sense to unlock dev. dependency versions?

Reply:
Yes. For different versions of python, you need to install different versions of dev packages.